README.md

```diff
@@ -1,3 +1,4 @@
 # rultor-python
+
```

Comment:
@aashutoshrathi Please put this in a separate commit before the main commit. It's necessary to make this PR pass. But it's actually not part of adding a `.coala` file and a `coala` test step. Just to keep things clean

Reply:
I changed the PR message to 
```
Created .coafile file and added bears
Fixed CI integration with coala.
Fixed errors raised by bears.
```